prow/cluster/jobs/istio/tools/istio.tools.yaml

```diff
@@ -80,3 +80,35 @@ presubmits:
                 value: "on"
         nodeSelector:
           testing: test-pool
+
+postsubmits:
+
+  istio/tools:
+
+    - name: tools-push-build-tools-image
+      branches:
+        - ^master$
+      cluster: test-infra-trusted
+      run_if_changed: '^docker/build-tools/'
+      annotations:
+        testgrid-dashboards: "tools"
```

Comment:
you'll need to add this dashboard to `testgrid/default.yaml` for this to work. The process isn't fully automated yet though so next time someone updates it they can just add it when needed